### DISCUSSION TITLE
Contributor call: 10 June 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#4157] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: June 10 2023 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jun-10-2023/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda
- Update on translation woes
- #4255 Proposal for how to handle translations in designs
- Social media and the lack thereof
- The upcoming newsletter
- TBD (comment below)

**Comment by tangerineshark** [comment]
This is me reminding myself that I tabled this agenda item from the last Contrib Call:

- Social media and the lack thereof

**Comment by bobgeorgethe3rd** [comment]
Snap pctBased options, what is the intended round behaviour? #4236 4236

**Reply by joostdecock** [reply]
Not adding this to the agenda given that the assumption that the documentation does not clarify this is incorrect: https://v2.freesewing.dev/reference/api/config/options/pct/snap#how-snapped-percentage-options-work

I will also update #4236 with the same link

**Comment by tangerineshark** [comment, marked as answer]
### Notes

Welcome new contributor call attendees Dave (braveness23) and Ju!

Agenda:

Update on translation woes

- Background: @joostdecock changed a filename, and it had a lot of knock-on effects on translation, causing us to hit our github rate limits.
- Crowdin also let us knwo we hit our limit and sent a threatening email, but they later got back to us and it's okay.
- Moral of the story: don't change filenames that affect tons of stuff (like "patterns" to "designs") without being sure it's worth it. And then probably give Joost a heads up.
- Related note: Crowdin always changes `---` in frontmatter to `- - -` and Joost is working on it.

https://github.com/freesewing/freesewing/pull/4255 Proposal for how to handle translations in designs

- Well first you need to know that i18n = "international translation" (like a11y is used for "accessibility").
- Currently, translation is added, but never removed, because, essentially, it's not connected in such a way that the translated content knows what's been removed.
- Proposal: Let's bring translation "closer" to what's actually using it (e.g., the login page), then we can more easily delete unused content. For example, a pattern and its translation should live side-by-side. If you want the technical details on what this means, pretty sure @joostdecock, @eriese, and @woutervdub knew what was up in more detail.
- @woutervdub asks how you prevent the same thing being translated multiple ways, and it is share that that's probably okay in that it's already a concern with the current system, but we're still chugging along.
- Note: `merge-i18n` (@joostdecock is this written right?) lets you pull existing translations for patterns inheriting from other patterns

Social media and the lack thereof

- We have some social media platforms that are basically un-managed right now (Instagram, Reddit, probably others...)
- We could use some folks who'd be game to help out with this and run specific platforms
- There was some debate about how much guidance/parameters/rules should be provided for potential recruits
- @tangerineshark is gonna write a newsletter blurb calling for support, and bounce it off @bobgeorgethe3rd

The upcoming newsletter: What's in it? What should be in it? Who's writing this stuff?

- @lexander2002's putting an article together on a workshop he led
- We've got an interview with Discord champion Popeye
- @tangerineshark and @bobgeorgethe3rd writing a call for social media contributors
- We're not ready to roll out an alpha of v3 yet, but can totally share some cool updates designed for v3, including designs slated for release (@BenJamesBen has the full list of these, I think), new features, and the Titan angled waist (@joostdecock is probably unavoidably on the hook for at least some of this - sorry, bud)

As always, blame mistakes or lack of clarity on @tangerineshark ! ❤️ 🍊 🦈

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
